example/spec/parallel/distributionAPI/distributionTEASpec.js

```diff
@@ -24,7 +24,7 @@ const s3Data = [
   '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf.met',
 ];
 
-xdescribe('TEA Distribution API', () => {
+describe('TEA Distribution API', () => {
```

Comment:
not related to this ticket.  But TEA is still deployed and support, and we want to enable the test